src/StockLocationInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
This is change needs highlighting as it shows how we are not only adding location as entities in the local stock implementation but making the stock manager aware of those using this interface. Great work